src/backend/core/country/views.py

```diff
@@ -42,6 +44,10 @@ class CountryListView(ListCreateAPIView):
 
     serializer_class = CountrySerializer
 
+    @check_user_is_staff_decorator()
```

Comment:
Please remove this decorator, it's used on both storefront & dashboard - it's just a GET view.

Reply:
Fixed.